flags_test.go

```diff
@@ -184,6 +185,7 @@ func (t *FlagsTest) DecimalNumbers() {
 		"--stat-cache-capacity=8192",
 		"--max-idle-conns-per-host=100",
 		"--max-conns-per-host=100",
+		"--kernel-dir-cache-ttl-secs=234",
```

Comment:
Can we also tests in this file for other cases of kernel-dir-cache-ttl-secs, e.g. -1, <-1, and `kernel-dir-cache-ttl-secs`>`max supported value of kernel-dir-cache-ttl-secs`